cli.sh

```diff
@@ -661,10 +661,10 @@ manage_trusted_repo() {
     if [ "$1" = "help" ]; then
         print_help_header
         echo "
-git hooks trust
-git hooks trust [revoke]
-git hooks trust [delete]
-git hooks trust [forget]
+git hooks trust [--local|--global]
```

Comment:
Can you please make this assume `--local` when not given to keep the previous behaviour?